app/src/main/java/org/sopt/at/screen/SearchScreen.kt

```diff
@@ -1,10 +1,49 @@
 package org.sopt.at.screen
 
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
-import androidx.compose.ui.graphics.Color
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.sopt.at.component.CommonTextField
+import org.sopt.at.ui.theme.TivingTheme
+import org.sopt.at.viewmodel.SearchViewModel
 
 @Composable
-fun SearchScreen() {
-    Text(text = "SEARCH", color = Color.White)
+fun SearchScreen(viewModel: SearchViewModel = remember { SearchViewModel() }) {
```

Comment:
```suggestion
fun SearchScreen(viewModel: SearchViewModel = viewModel()) {
    ...
}
```
이런 식으로 viewModel()을 사용해 생명주기에 안전하게 연동하는 방법은 어떨까요!?